.github/workflows/appsec.yml

```diff
@@ -184,26 +184,20 @@ jobs:
 
   # Same tests but on the official golang container for linux
   golang-linux-container:
-    name: golang-containers ${{ toJSON(matrix) }}
-    runs-on: ubuntu-latest-16-cores
+    name: ${{ matrix.platform }} golang:${{ matrix.go-version }}-${{ matrix.distribution }}
+    # We use ARM runners when needed to avoid the performance hit of QEMU
+    runs-on: ${{ matrix.platform == 'linux/amd64' && 'ubuntu-latest-16-cores' || 'arm-4core-linux' }}
```

Comment:
Works for me to have this indirect definition of `runs-on`. I say so because you went explicit in go-libddwaf with directly listing in the matrix the runner label to iterate on.

Reply:
These are actually two different cases... in `go-libddwaf`, this is running directly on the runner (e.g: the runner is also the image definition), whereas here this is all containerized, so the runner & the image definition are correlated but separate.